mlflow/tensorflow.py

```diff
@@ -175,18 +175,18 @@ def load_model(model_uri, tf_sess):
     """
     Load an MLflow model that contains the TensorFlow flavor from the specified path.
 
-    **This method must be called within a TensorFlow graph context.**
+    *This method must be called within a TensorFlow graph context.*
 
-    :param model_uri: The location, in URI format, of the MLflow model, for example:
+    :param model_uri: The location, in URI format, of the MLflow model. For example:
 
                       - ``/Users/me/path/to/local/model``
                       - ``relative/path/to/local/model``
                       - ``s3://my_bucket/path/to/model``
                       - ``runs:/<mlflow_run_id>/run-relative/path/to/model``
 
-                      For more information about supported URI schemes, see the
-                      `Artifacts Documentation <https://www.mlflow.org/docs/latest/tracking.html#
-                      supported-artifact-stores>`_.
+                      For more information about supported URI schemes, see
+                      `Artifact Stores <https://www.mlflow.org/docs/latest/tracking.html#
```

Comment:
Based on above suggestion about renaming the `Artifact Store Locations` to `Referencing Artifacts`, this link should be renamed.